src/System.Composition.Convention/src/System/Composition/Convention/ConventionBuilder.cs

```diff
@@ -288,8 +288,8 @@ private List<Attribute> ReadMemberCustomAttributes(Type reflectedType, System.Re
         /// <returns>The list of applied attributes.</returns>
         public override IEnumerable<Attribute> GetCustomAttributes(Type reflectedType, System.Reflection.ParameterInfo parameter)
         {
-            Requires.NotNull(parameter, "reflectedType");
-            Requires.NotNull(parameter, "parameter");
+            Requires.NotNull(reflectedType, nameof(reflectedType));
```

Comment:
While I'm sure this is what was originally intended, are we going to start failing in places we didn't previously?